javascript-methods/main.js

```diff
@@ -0,0 +1,73 @@
+const numberOne = 7;
+const numberTwo = 10;
+const numberThree = 35;
+const maximumValue = Math.max(numberOne, numberTwo, numberThree);
+
+console.log('maximumValue:', maximumValue);
+
+const heroes = ['Superman', 'Batman', 'Robin', 'Green Arrow'];
+
+let randomNumber = Math.random(heroes);
+randomNumber = randomNumber * heroes.length - 1;
+
+const randomIndex = Math.floor(randomNumber);
+
+console.log('randomIndex:', randomIndex);
+
+const randomHero = heroes[randomIndex];
+
+console.log('randomHero:', randomHero);
+
+const arr = [
+  {
+    title: 'Holes',
+    author: 'Louis Sachar',
+  },
+  {
+    title: 'The Making of the Atomic Bomb',
+    author: 'Richard Rhodes',
+  },
+  {
+    title: 'Bugs Journal',
+    author: 'Bug',
+  },
+];
+
+const library = arr;
```

Comment:
I would just initialize `library` on line 21 rather than creating `arr` and then assigning it to `library`. It's not needed. Just have an array assigned to your `library` variable